tests/cupy_tests/binary_tests/test_packing.py

```diff
@@ -39,3 +40,17 @@ def test_unpackbits(self):
         self.check_unpackbits([1])
         self.check_unpackbits([255])
         self.check_unpackbits([100, 200, 123, 213])
+        a = cupy.array([10, 20, 30])
+        pytest.raises(TypeError, cupy.unpackbits, a)
+        pytest.raises(TypeError, cupy.unpackbits, a.astype(float))
+
+    def test_pack_unpack_order(self):
+        for bo in ['big', 'little']:
+            self.check_unpackbits([], bitorder=bo)
+            self.check_unpackbits([0], bitorder=bo)
+            self.check_unpackbits([1], bitorder=bo)
+            self.check_unpackbits([255], bitorder=bo)
+            self.check_unpackbits([100, 200, 123, 213], bitorder=bo)
+        a = cupy.array([10, 20, 30], dtype=cupy.uint8)
+        pytest.raises(ValueError, cupy.unpackbits, a, bitorder='r')
```

Comment:
can you separate these two assers into a new test?

`def test_unpack_invalid_order` or something similar :).
Thanks!

Reply:
Thank you for the suggestion. I did the same for the two asserts I've added to `test_unpackbits` in order to assert the array validity.